keps/sig-network/563-dual-stack/README.md

```diff
@@ -1345,109 +1389,135 @@ This capability will move to stable when the following criteria have been met.
   - [X] Feature gate (also fill in values in `kep.yaml`)
     - Feature gate name: IPv6DualStack
     - Components depending on the feature gate: kube-apiserver, kube-controller-manager, kube-proxy, and kubelet
-  - [ ] Other
-    - Describe the mechanism:
-    - Will enabling / disabling the feature require downtime of the control
-      plane?
-    - Will enabling / disabling the feature require downtime or reprovisioning
-      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
 * **Does enabling the feature change any default behavior?**
-  No. Pods and Services will remain single stack until cli flags have been modified
-  as described in this KEP. Once modified, existing and new services will remain
-  single stack until user requests otherwise. Pods will become dual-stack however
-  it will maintain the same ipfamily used in before enabling feature flag.
+  Pods and Services will remain single-stack until cli flags have been modified
+  as described in this KEP. Existing and new services will remain single-stack
+  until user requests otherwise. Pods will become dual-stack once CNI is
+  configured for dual-stack.
```

Comment:
A little confusing: 

> Pods and Services will remain single-stack until cli flags have been modified
  as described in this KEP.

and

> Pods will become dual-stack once CNI is configured for dual-stack.

are contradictory. I think the distinction is api vs pod runtime environment. Please clear that up.

Reply:
Thanks - edited.